src/EditorFeatures/VisualBasicTest/Classification/SemanticClassifierTests.vb

```diff
@@ -543,5 +543,21 @@ end sub
 
             Await TestInClassAsync(text)
         End Function
+
+        <WorkItem(14688, "https://github.com/dotnet/roslyn/issues/14688")>
+        <Fact, Trait(Traits.Feature, Traits.Features.Classification)>
+        Public Async Function OperatorsWithDefaultValuesAreNotAllowed() As Task
+            Dim text =
+<code>
+Public Class Foo
+    Public Shared Operator Not(Optional x As Foo) As Foo
+    End Operator
+End Class
+</code>.NormalizedValue()
+
+            Await TestAsync(text,
```

Comment:
Agreed.  Lots of stuff isn't legal in the language (for example, an optional parameter preceding a required parameter).  But we still get the experience right even in those cases.